dotBloch/Assets/Classes/Constants.cs

```diff
@@ -0,0 +1,28 @@
+﻿class Constants
+{
+    public static class symbol{
+
+        public const string bloch_sphere = "|Ψ> = ";
+        public const string quantum_one = " |1>";
+        public const string quantum_zero = " |0> ";      
+    }
+
+    public static class error{
+        public const string angle_is_wrong = "Value of theta or phi angle is wrong";
+    }
+
+    public static class character{
+        public const string space = " ";
+        public const string dot = ".";
+        public const string comma = ",";
+
+        public const string rounding_header = "N";
+    }
+
+    public static class math{

```

Comment:
Jak na to spojrzałem to pomyślałem sobie o jakiś stałych z takimi rzeczami o ile mogą być stringami